angelssmelting/prototypes/override/smelting-override-alloy-support.lua

```diff
@@ -46,30 +46,32 @@ end
 if mods["bobassembly"] and settings.startup["bobmods-assembly-multipurposefurnaces"].value then
   -- keep metal mixing furnaces around
   OV.add_unlock("automation", "stone-mixing-furnace")
+  OV.patch_recipes({
+    {
+      name = "electric-chemical-mixing-furnace",
+      ingredients = {
+        {"electric-mixing-furnace", "electric-furnace"}
+      }
+    }
+  })
 
-  --update localisation strings
-  local replace = {
-    {old = "stone-mixing-furnace", new = "Stone multi-ingredient furnace"},
-    {old = "steel-mixing-furnace", new = "Steel multi-ingredient furnace"},
-    {old = "electric-mixing-furnace", new = "Electric multi-ingredient furnace"},
-    {old = "electric-chemical-mixing-furnace", new = "Electric multi-ingredient furnace 2"},
-    {old = "electric-chemical-mixing-furnace-2", new = "Electric multi-ingredient furnace 3"},
-  }
-  --add oil furnace to table if setting active
-  if settings.startup["bobmods-assembly-oilfurnaces"] then
-    table.insert(replace,{old = "fluid-mixing-furnace", new = "Fluid burning multi-ingredient furnace"})
+  --update metal mixing furnace localisation
+  for _,rep in pairs({
+    {name = "stone-mixing-furnace", locale = "angels-stone-ingredient-furnace"},
+    {name = "steel-mixing-furnace", locale = "angels-steel-ingredient-furnace"},
+    {name = "electric-mixing-furnace", locale = "angels-electric-ingredient-furnace-1"},
+    {name = "electric-chemical-mixing-furnace", locale = "angels-electric-ingredient-furnace-2"},
+    {name = "electric-chemical-mixing-furnace-2", locale = "angels-electric-ingredient-furnace-3"},
+    settings.startup["bobmods-assembly-oilfurnaces"] and {name = "fluid-mixing-furnace", locale = "angels-fluid-ingredient-furnace"} or nil,
```

Comment:
Should be `settings.startup["bobmods-assembly-oilfurnaces"].value`

Reply:
yes it should, but it still worked... either way, will fix this one quickly.

Reply:
Without the change it would crash if fluid furnaces was disabled. Trying to set the localised_name of a machine that didn't exist. Looks good now.